workflow/common/util_notwin.go

```diff
@@ -0,0 +1,10 @@
+// +build !windows
+
+package common
+
+// RunShellCommand is a convenience function to use RunCommand for shell executions. It's os-specific
+// and runs `cmd` in windows (see `util_windows.go`)
+func RunShellCommand(arg ...string) ([]byte, error) {
```

Comment:
For simplicity instead of build flags I would determine the OS at runtime via:
```golang
if runtime.GOOS == "windows" {
    // use cmd /c instead of sh -c
}
```

Then we could just move this function to `util.go` and don't need two additional files.